app/flatpak-cli-transaction.c

```diff
@@ -506,8 +506,6 @@ operation_error (FlatpakTransaction            *transaction,
     msg = g_strdup_printf (_("%s%s%s already installed"), on, flatpak_ref_get_name (rref), off);
   else if (g_error_matches (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED))
     msg = g_strdup_printf (_("%s%s%s not installed"), on, flatpak_ref_get_name (rref), off);
-  else if (g_error_matches (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED))
-    msg = g_strdup_printf (_("%s%s%s not installed"), on, flatpak_ref_get_name (rref), off);
```

Comment:
maybe this if-else cascade should just be a switch instead ?

Reply:
After re-implementing it that way, it seems cleaner in some ways and worse in other ways. See the new commit and what do you think?

Reply:
Yeah, not much better indeed. Keeping the "Error" and "Warning" prefixes translated separately would help for that, but ruin the intention of this PR, so ... not sure.

Reply:
Dropping that commit, as discussed in person